src/settings/Settings.ts

```diff
@@ -279,6 +281,15 @@ export class MediaDbSettingTab extends PluginSettingTab {
 					void this.plugin.saveSettings();
 				});
 			});
+		new Setting(containerEl)
+			.setName('Embed posters in YAML')
+			.setDesc('Embed the poster urls as images in the yaml to integrate with dataview.')
```

Comment:
This pretty much sums it up nicely and no other way to put it,  just adding a alternative

"Embed the URLs of the poster images within the image property using the image embedding syntax `![](URL)`. This will enable the images to directly integrate with Dataview, as opposed to simply being a hyperlink."

Reply:
> Save image links as markdown embeds in the frontmatter, for easier handling with e.g. Dataview.